pkg/nimble/doc.txt

```diff
@@ -1,10 +1,10 @@
 /**
- * @defgroup pkg_nimble     NimBLE BLE stack
+ * @defgroup ble_nimble NimBLE
  * @ingroup  pkg
- * @ingroup  net
+ * @ingroup  net_ble
```

Comment:
`ble`

Reply:
Then again, why not prefix everything with `net_` to stick to the common theme?

Reply:
ups